hrm_oo.py

```diff
@@ -0,0 +1,44 @@
+import csv
+import numpy as np
+from readHR import EcgReader
+from hrmcalcs2oo import hrmcalcs
+from hrmtb import TachyBrady
+
+
+class HrmOutput:
+
+    def __init__(self, file='Test_ECG.csv', start_min=3, end_min=5,
```

Comment:
Be sure to have a docstring that describes these input params (and include units!).